util/useragentutil/useragentutil.go

```diff
@@ -0,0 +1,82 @@
+package useragentutil
+
+import (
+	"strings"
+
+	"github.com/mileusna/useragent"
+)
+
+// Device types enum
+const (
+	DEVICE_DESKTOP = iota
+	DEVICE_MOBILE
+	DEVICE_TABLET
```

Comment:
Nitpick: Following Go naming standards, use MixedCaps here instead of SCREAMING_CAPS.

Reply:
Updated